### PR TITLE
Note that EFS encryption is now available and small adjustments.

### PR DESCRIPTION
**Pull request by bgdnlp**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -944,10 +944,9 @@ EFS
 ### EFS Basics
 
 -	📒 [Homepage](https://aws.amazon.com/efs/) ∙ [User guide](http://docs.aws.amazon.com/efs/latest/ug) ∙ [FAQ](https://aws.amazon.com/efs/faq/) ∙ [Pricing](https://aws.amazon.com/efs/pricing/)
--	🐥**EFS** is Amazon’s new (general release 2016) network filesystem.
+-	🐥**EFS** is Amazon’s network filesystem. It’s presented as an [NFSv4.1](https://en.wikipedia.org/wiki/Network_File_System#NFSv4) server. Any compatible NFS client can mount it.
```
Comment (QuinnyPig) [inline README.md]: Might want to explicitly call out that any NVSv4 client can mount it. Windows doesn't support this the last time I checked, and that's caught some people out.

File: README.md [hunk 2]
```diff
@@ -972,7 +971,7 @@ EFS
 ### EFS Gotchas and Limitations
 
 -	🔸 A number of NFSv4.1 features are [not supported](http://docs.aws.amazon.com/efs/latest/ug/nfs4-unsupported-features.html) and there are some [limits](http://docs.aws.amazon.com/efs/latest/ug/limits.html) to the service.
--	🔸 As of 2017-02, EFS does not offer disk level encryption, though it is on the roadmap.
+-	🔸 As of 2017-08, EFS offers disk level encryption for new drives. For file systems created before that date, encryption can only be achieved by moving the data to a new EFS.
```
Comment (QuinnyPig) [inline README.md]: Just stylistic-- I'd throw the word "volume" at the end of the sentence.

File: README.md (hunk 1 above)
Comment (bgdnlp) [inline README.md]: Done. Lack of Windows support is specifically called out on another line, left it out here.

**Comment by QuinnyPig** [comment]
Thanks! Looks great.

**Pull request merged**